### DISCUSSION TITLE
Separate "natural spawn" from "breeding"

### DISCUSSION THREAD

**Discussion by WarnDa** [BG-Software-LLC/SuperiorSkyblock2#680] [Denied Feature Requests] [answered]
Currently you can't reproduce animals by breeding them if "natural animal spawn" is OFF.

It would be could to separate the two option in '/is settings'
because player don't want natural mob to spawn, but want to reproduce their actual mob stacks.
(I'm using wildstacker to stack mobs)

**Comment by OmerBenGera** [comment, marked as answer]
Natural mob spawning still allows breeding to work.
If that's not the case for you, open an issue.
